### ISSUE TITLE
payloads

### ISSUE THREAD

**Issue by 1br00** [gauravnarwani97/Trishul#7]
hello bro where is payloads which are injected here, can i add my list of payloads please?

**Comment by M3hank** [comment]
> hello bro where is payloads which are injected here, can i add my list of payloads please?

I don't think so there is a list of payload or something, like in xss it checks reflection of some special character like "<" & ">".
